packages/storybook/src/stories/grid-toolbar.stories.tsx

```diff
@@ -61,3 +62,30 @@ export const CsvExport = () => {
     </div>
   );
 };
+export const Test = () => {
```

Comment:
Can we have a proper story name please?

Reply:
That one is not needed, I forgot to remove it.